net/nimble/controller/src/ble_ll_scan.c

```diff
@@ -1677,7 +1682,11 @@ ble_ll_scan_get_aux_data(struct ble_ll_scan_sm *scansm,
         (*aux_data)->chan = tmp_aux_data.chan;
         (*aux_data)->offset = tmp_aux_data.offset;
         (*aux_data)->mode = tmp_aux_data.mode;
-
+        if (has_addr) {
```

Comment:
this is allowed only on 1M and RFU on other PHYs

and we should probably handle TargetA in same way

Reply:
Yes, there is Jira on this.